src/Utils/generics.ts

```diff
@@ -316,6 +316,7 @@ export const generateMdTagPrefix = () => {
 }
 
 const STATUS_MAP: { [_: string]: proto.WebMessageInfo.Status } = {
+	'sender': proto.WebMessageInfo.Status.SERVER_ACK,
```

Comment:
I think `sent` makes more sense.

```suggestion
	'sent': proto.WebMessageInfo.Status.SERVER_ACK,
```

Reply:
Or: `server-received`

Reply:
I think it doens't work because 'sender' comes from the `MessageReceiptType` type:

`export type MessageReceiptType = 'read' | 'read-self' | 'hist_sync' | 'peer_msg' | 'sender' | 'inactive' | 'played' | undefined`

I would have to change some more code...

Reply:
You can find that out putting some log in to line 628, file `src/Socket/messages-recv.ts`

```typescript
try {
  await Promise.all([
    processingMutex.mutex(
	async() => {
	  logger.debug(attrs.type) // prints 'sender'

	  const status = getStatusFromReceiptType(attrs.type)
	  ...
```